SUPPORT.md

```diff
@@ -0,0 +1,5 @@
+# Support
+
+If you encounter an issue with Site Kit, first check our [troubleshooting guide](https://sitekit.withgoogle.com/documentation/fix-common-issues/). For plugin usage questions and compatibility issues with other plugins, please use the [plugin's support forum](https://wordpress.org/support/plugin/google-site-kit). Before opening a new topic, please search the forum for existing topics as someone else has likely reported the issue already.
+
+Find documentation and a blog with news on the [plugin website](https://sitekit.withgoogle.com).
```

Comment:
@felixarntz "a blog" sounds a bit off here. How about this?

```suggestion
Find documentation and news on [the official Site Kit website](https://sitekit.withgoogle.com).
```

Reply:
LGTM!